.github/workflows/release.yml

```
@@ -0,0 +1,46 @@
+on:
+  release:
+    types:
+      - published
+
+name: release
+
+# Needed to access the workflow's OIDC identity.
+permissions:
+  id-token: write
+
+jobs:
+  pypi:
+    name: Build, sign and publish release to PyPI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+
+    - name: deps
+      run: python -m pip install -U build
+
+    - name: build
+      run: python -m build
+
+    - name: sign
+      run: |
+        # use the latest development changes for sigstore, at least
+        # until things are stabilized further
+        make dev && source env/bin/activate
```

This could also be:
```suggestion
        python -m pip install .
```
Not sure we need to invoke the entire development environment and it's dependencies here.

---

Good point, force of habit 🙂